third_party/arcs/examples/BUILD

```diff
@@ -15,3 +15,5 @@ arcs_manifest_proto(
     src = "VisionPipeline.arcs",
     deps = [],
 )
+
+licenses(["notice"])
```

Comment:
Why are the `licenses(` line at the end? Why not add after the copyright block on the top?

Reply:
I used a shell script to add `licenses` to each file to reduce toil. I was hoping that `buildifier` would fix the placement, but it did not. I can take another pass at the shell script to put it at the top; even if `buildifier` doesn't fix that, it's at least a more natural place to find it.

Reply:
Actually, there are only 18 files. I'll just do it by hand.

Reply:
Fixed this. Also fixed the copyright headers in these files and added a presubmit check for this for `BUILD` files only; we should add it for more files in the future.